examples.md

```diff
@@ -200,40 +197,12 @@ We cache the elements of the Cabal store separately, as the entirety of `~/.caba
 
 ## Node - npm
 
-For npm, cache files are stored in `~/.npm` on Posix, or `%AppData%/npm-cache` on Windows. See https://docs.npmjs.com/cli/cache#cache
+For npm, cache files are stored in `~/.npm` on Posix, or `%AppData%/npm-cache` on Windows, but it's possible to use `npm config get cache` to find the path on any platform. See https://docs.npmjs.com/cli/cache#cache for more details.
 
 If using `npm config` to retrieve the cache directory, ensure you run [actions/setup-node](https://github.com/actions/setup-node) first to ensure your `npm` version is correct.
 
 >Note: It is not recommended to cache `node_modules`, as it can break across Node versions and won't work with `npm ci`
 
-### macOS and Ubuntu
```

Comment:
IMO, we should keep this example as well, as it is much more simple without using an extra step. You can instead combine the two headers below (Windows and multiple systems) into one, which will be a much better approach. @bishal-pdMSFT  thoughts?

Reply:
It's slightly simpler, but this is just copying and pasting anyway. Personally as a user, I just want one example that works everywhere.

More importantly though, this example isn't even 100% correct on Linux & Mac, as the hardcoded path here is actually configurable (by local & global config or env var). Checking `npm config get cache` is the only way to know what the correct path is for sure on any platform.

Also every other Node package manager in this file (Lerna, Yarn, Yarn 2) already uses a single two-step example like this, instead of having a special case that hardcodes the Mac/Linux default path.